src/realm/collection_list.cpp

```diff
@@ -293,4 +321,52 @@ void CollectionList::set_collection_ref(Index index, ref_type ref)
     }
 }
 
+auto CollectionList::get_index(size_t ndx) const noexcept -> Index
```

Comment:
No so sure why we need auto return type here.

Reply:
It felt good at the time. Alternatively we should have used `CollectionParent::Index` which requires more typing.